etc/workflows/studio-upload.xml

```diff
@@ -0,0 +1,448 @@
+            } else if (!$(this).is(':checked')) {
+              awsWasChecked = $('#publishToAws').prop('checked');
+              engageWasChecked = $('#publishToEngage').prop('checked');
+
+              $('#publishToAws').prop('disabled', true);
+              $('#publishToEngage').prop('disabled', true);
+              $('#publishToAws').prop('checked', false);
+              $('#publishToEngage').prop('checked', false);
+            }
+        });
+      </script>
+    ]]>
+  </configuration_panel>
+
+  <operations>
+
+    <!-- Apply the default workflow configuration -->
+
+    <operation
+        id="defaults"
+        description="Applying default configuration values">
+      <configurations>
+        <configuration key="skipEditorPreview">true</configuration>
+        <configuration key="publishToEngage">true</configuration>
+        <configuration key="publishToApi">true</configuration>
+        <configuration key="publishToOaiPmh">false</configuration>
+        <configuration key="publishToAws">false</configuration>
+        <configuration key="thumbnailType">0</configuration>
+        <configuration key="thumbnailPosition">1</configuration>
+      </configurations>
+    </operation>
+
+    <operation
+        id="inspect"
+        exception-handler-workflow="partial-error"
+        description="Inspecting audio and video streams">
+      <configurations>
+        <configuration key="overwrite">false</configuration>
+        <configuration key="accept-no-media">false</configuration>
+      </configurations>
+    </operation>
+
+    <operation
+        id="series"
+        exception-handler-workflow="partial-error"
+        description="Applying series dublincore catalogs and access control entries">
+      <configurations>
+        <!-- <configuration key="series">[put your fixed series id here]</configuration> -->
+        <configuration key="attach">*</configuration>
+        <configuration key="apply-acl">true</configuration>
+      </configurations>
+    </operation>
+
+    <operation
+        id="tag"
+        exception-handler-workflow="partial-error"
+        description="Tagging source material for archival">
+      <configurations>
+        <configuration key="source-flavors">dublincore/*,security/*,*/source</configuration>
+        <configuration key="target-tags">+archive</configuration>
+      </configurations>
+    </operation>
+
+    <operation
+        id="include"
+        description="Include studio tracks normalization tasks">
+      <configurations>
+        <configuration key="workflow-id">partial-transcode-studio-tracks</configuration>
+      </configurations>
+    </operation>
+
+    <operation
+        id="snapshot"
+        fail-on-error="true"
+        exception-handler-workflow="partial-error"
+        description="Archiving">
+      <configurations>
+        <configuration key="source-tags">archive</configuration>
+      </configurations>
+    </operation>
+
+    <operation
+        id="prepare-av"
```

Comment:
Just wondering if you couldn't do the initial re-encoding in here instead of adding another `encode` operation?

Reply:
This operations will be skipped in most cases. Opencast studio pack the audio stream in both video files on dualstream recording. But just in case…